src/components/CCPForm/CCPFormPage.tsx

```diff
@@ -180,25 +160,27 @@ const CCPFormPage = ({
   );
 
   return (
-    <div className={classes.ccpWrapper}>
-      <div className={classes.ccpFormTopSection}>
-        <div className={classes.ccpHeader}>
-          <Typography variant="h3">
-            {mode === 'new' ? 'Create a CCP' : 'Edit CCP'}
-          </Typography>
-          <div className={classes.ccpBtnPosition}>
-            <Button
-              color="primary"
-              className={classes.ccpCancelBtn}
-              onClick={handleCancel}
-            >
-              Cancel
-            </Button>
-          </div>
-        </div>
-      </div>
-      <div className={classes.ccpForm}>{content}</div>
-    </div>
+    <Box minHeight="100vh">
```

Comment:
I wanted to bring this to your attention, just in case it wasn't intentional.

Just to confirm, if the components on the page ever overflow, we'd get a scroll bar. Was this intentional?

If not, maybe just change this to `height="100vh"` instead of `minHeight`

Reply:
Yeah, I did it to stay consistent with `EventCreationPage.tsx` but I think changing it to `height="100vh"` makes sense.

Reply:
Yeah I wrote `minHeight` to stay consistent with EventCreationPage.tsx. Would it make sense to make the change to `height="100vh"` on that page as well?

Reply:
hmm if you're being consistent it should be fine. I think `height` and `minHeight` should both be ok, but we should test it on different screen sizes just to be sure